charts/fpl-case-service/charts/fpl-prd-mock/values.yaml

```diff
@@ -0,0 +1,7 @@
+users:
+  - email: damian@swansea.gov.uk
+    identifier: 365465
+  - email: sam@hillingdon.gov.uk
+    identifier: c9166989-84e6-45fe-94c8-0148f908f35b
+prd:
+  url: http://rd-professional-api-aat.service.core-compute-aat.internal
```

Comment:
Couldn't it use the value from the config somehow? What if we wanted to use the mock on different envs? How could we configure that?

Reply:
done, move to parent chart values file for preview env

Reply:
still users are aat specific, do you want them to be configured per env?